pkg/mqtt/server.go

```diff
@@ -408,7 +408,11 @@ func (s *Server) listenToBlocks(ctx context.Context) error {
 }
 
 func (s *Server) listenToAcceptedBlocksMetadata(ctx context.Context) error {
```

Comment:
merge these methods, else you will be listening to the same stream multiple times. And this will generate a lot of load because it is not streaming all the blocks and its metadata changes